tests/output-eve-anomaly/test.yaml

```diff
@@ -0,0 +1,26 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+
+checks:
+  - filter:
+      filename: eve.json
```

Comment:
no need to have this filename setting, eve.json is the default

Reply:
Will remove.